server/src/test/java/keywhiz/service/resources/automation/v2/SecretResourceTest.java

```diff
@@ -66,6 +68,23 @@
   // createSecret
   //---------------------------------------------------------------------------------------
 
+  @Test public void createSecretWithOwner() throws Exception {
+    String secretName = UUID.randomUUID().toString();
+    String ownerName = UUID.randomUUID().toString();
+
+    assertEquals(201, createGroup(ownerName).code());
+
+    CreateSecretRequestV2 request = CreateSecretRequestV2.builder()
+        .name(secretName)
+        .owner(ownerName)
+        .content(encode("foo"))
+        .build();
+    create(request);
+
+    SecretDetailResponseV2 details = lookup(secretName);
+    assertEquals(ownerName, details.owner());
+  }
```

Comment:
This test verifies that we can create a secret with an owner via the V2 automation API.